docs/Contributing/Testing-and-local-development.md

```diff
@@ -489,7 +489,9 @@ FLEET_SERVER_SANDBOX_ENABLED=1 FLEET_PACKAGING_GLOBAL_ENROLL_SECRET=xyz  ./build
 Be sure to replace the `FLEET_PACKAGING_GLOBAL_ENROLL_SECRET` value above with the global enroll
 secret from the `fleetctl package` command used to build the installers.
 
-MinIO also offers a web interface at http://localhost:9001. Credentials are `minio` / `minio123!`.
+MinIO also offers a web interface at http://localhost:9001. Credentials are `minio` / `minio123!`. When starting the
+Fleet server up with `--dev` the server will look for installers in the `software-installers-dev` MinIO bucket. You can
+create this bucket via the MinIO web UI (it is *not* created by default when setting up the docker-compose environment).
```

Comment:
Actually I fixed this a few hours ago on this PR: https://github.com/fleetdm/fleet/pull/21650/files#diff-92cebe996f3c912cf7faa9227ee3fa22fc6b6d225f6628bb6b9c277b54e1dbb3R129-R131

(When running with `serve` --dev it will create the bucket if it doesn't exist.)

Reply:
Nice! Can revise this once that's merged in then.